README.md

```diff
@@ -457,6 +457,8 @@ To rebalance **all** topics in a cluster, use the `rebalance` subcommand, which
 function on all qualifying topics. It will inventory all topic configs found at  `--path-prefix` for a cluster
 specified by `--cluster-config`.
 
+To rebalance topics in a cluster that's missing any topic configs, use the `rebalance` subcommand with the `--bootstrap-missing-configs` flag. This will temporarily bootstrap any missing topic configs at `--path-prefix`.
```

Comment:
```suggestion
To rebalance topics in a cluster that exist without topic configuration files, use the `rebalance` subcommand with the `--bootstrap-missing-configs` flag. This will temporarily bootstrap any missing topic configs at `--path-prefix`. This can also be used to use topicctl as a topic rebalancing tool, without using its topic configuration management features
```